web-ng/src/app/services/feature/feature.service.ts

```diff
@@ -66,4 +68,28 @@ export class FeatureService {
   getSelectedFeature$(): Observable<Feature> {
     return this.selectedFeature$;
   }
+
+  addPoint(lat: number, lng: number): Promise<void> {
+    // TODO: Update to use `await firstValueFrom(getActiveProject$()` when
+    // upgrading to RxJS 7.
+    return this.projectService
+      .getActiveProject$()
+      .pipe(take(1))
+      .toPromise()
+      .then(project => this.addPointInternal(project, lat, lng));
+  }
+
+  private addPointInternal(project: Project, lat: number, lng: number) {
+    if (project.layers.isEmpty()) {
+      return Promise.resolve();
+    }
+    const newFeature = new Feature(
+      this.dataStore.generateId(),
+      project.layers.keySeq().first(),
```

Comment:
Can we add a comment or TODO here since adding to arbitrary layer is a temporary solution?

Reply:
Sorry, could have sworn I did this.. adding one now!